driver_cpl/cime_config/config_component.xml

```diff
@@ -544,7 +552,7 @@
   <entry id="BATCH_SYSTEM">
      <type>char</type>
      <default_value>none</default_value>
-     <valid_values>moab,pbs,lsf,slurm,cobalt,cobalt_theta,none</valid_values>
+     <valid_values>pbs,lsf,slurm,cobalt,cobalt_theta,none</valid_values>
```

Comment:
I think that you messed up the merge.  I'm pretty sure that the code you are replacing here is what we want and not the replacement.